behave.ini

```diff
@@ -0,0 +1,2 @@
+[behave]
+paths = ./tests/features
```

Comment:
For all your files check missing blank link at end of file

Reply:
done